docs/src/concepts.md

```diff
@@ -0,0 +1,116 @@
+
+In recent years, several leaders have been emerging as de facto standards with significant community uptake:
+
++ Kilosort
++ JRClust
++ MountainSort
++ SpyKING CIRCUS
+
+Kilosort provides most automation and has gained significant popularity, being adopted as one of the key spike sorting methods in the majority of the teams/collaborations we have worked with. As part of our Year-1 NIH U24 effort, we provide support for data ingestion of spike sorting results from Kilosort. Further effort will be devoted for the ingestion support of other spike sorting methods. On this end, a framework for unifying existing spike sorting methods, named [SpikeInterface](https://github.com/SpikeInterface/spikeinterface), has been developed by Alessio Buccino, et al. SpikeInterface provides a convenient Python-based wrapper to invoke, extract, compare spike sorting results from different sorting algorithms.
+
+## Key Partnerships
+
+Over the past few years, several labs have developed DataJoint-based data management and processing pipelines for Neuropixels probes. Our team collaborated with several of them during their projects. Additionally, we interviewed these teams to understand their experimental workflow, pipeline design, associated tools, and interfaces. These teams include:
+
+- [International Brain Lab](https://internationalbrainlab.org) - [https://github.com/int-brain-lab/IBL-pipeline](https://github.com/int-brain-lab/IBL-pipeline)
+
+- [Mesoscale Activity Project (HHMI Janelia)](https://github.com/mesoscale-activity-map) - [https://github.com/mesoscale-activity-map/map-ephys](https://github.com/mesoscale-activity-map/map-ephys)
+
+- Moser Group (Norwegian University of Science and Technology) - see [pipeline design](https://moser-pipelines.readthedocs.io/en/latest/ephys/overview.html)
+
+- Andreas Tolias Lab (Baylor College of Medicine)
+
+- BrainCoGs (Princeton Neuroscience Institute)
+
+- Brody Lab (Princeton University)
+
+## Element Architecture
+
+Each of the DataJoint Elements are a set of tables for common neuroinformatics modalities to organize, preprocess, and analyze data. Each node in the following diagram is either a table in the Element itself or a table that would be connected to the Element.
+
+![element-array-ephys diagram](https://raw.githubusercontent.com/datajoint/element-array-ephys/main/images/attached_array_ephys_element_acute.svg)
+
+### `subject` schema ([API docs](https://datajoint.com/docs/elements/element-animal/api/element_animal/subject))
+- Although not required, most choose to connect the `Session` table to a `Subject` table.
+
+| Table | Description |
+| --- | --- |
+| Subject | Basic information of the research subject |
+
+### `session` schema ([API docs](https://datajoint.com/docs/elements/element-session/api/element_session/session_with_datetime))
+
+| Table | Description |
+| --- | --- |
+| Session | Unique experimental session identifier |
+
+### `probe` schema ([API docs](../api/element_array_ephys/probe))
```

Comment:
please verify that this link works in local deployment

Reply:
It is working in local